src/Resources/public/js/order-edit.js

```diff
@@ -1,24 +1,32 @@
-window.addEventListener('DOMContentLoaded', (event) => {
-    document.querySelectorAll('button.delete-order-item').forEach((button) => {
-        button.addEventListener('click', (event) => {
-            const orderLineId = event.currentTarget.closest('tr').remove();
-        });
+document.querySelectorAll('button.delete-order-item').forEach((button) => {
+    button.addEventListener('click', (event) => {
+        var row = event.currentTarget.parentElement.parentElement;
```

Comment:
Use `event.currentTarget.closest('..')`

Reply:
<img width="472" alt="Zrzut ekranu 2024-06-27 o 11 24 57" src="https://github.com/Setono/sylius-order-edit-plugin/assets/6212718/d0548ad2-ec82-410e-9237-f53ef37b057f">
hmm, doesn't seems to work :/

Reply:
I don't understand how that's possible :D